src/utils/pageWrappers/withSentry.js

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect } from 'react'
+import { useAuthUser } from 'next-firebase-auth'
+import * as Sentry from '@sentry/node'
```

Comment:
Could you do me a favor and comment here and in `initSentry.js` with a reminder of how this works, given we tripped over it the other day? Something like:
```
// Our Webpack config swaps in the browser Sentry package when in
// a browser environment. See this example:
// https://github.com/vercel/next.js/blob/canary/examples/with-sentry/next.config.js#L45
```